### PR TITLE
Neo Roller

### PR DESCRIPTION
**Pull request by YoniKiriaty**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 1]
```diff
@@ -38,7 +36,7 @@ private OI() {
         mainJoystick = new SmartJoystick(RobotConstants.Joystick.MAIN);
         secondJoystick = new SmartJoystick(RobotConstants.Joystick.SECOND);
         initButtons();
-        initializeDefaultCommands();
+        //initializeDefaultCommands();
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/OI.java:39]: remove //

File: src/main/java/edu/greenblitz/robotName/Robot.java [hunk 2]
```diff
@@ -43,8 +43,8 @@ public void robotInit() {
         Pathfinding.setPathfinder(new LocalADStar());
         CommandScheduler.getInstance().enable();
         initializeLogger();
-        initializeSubsystems();
-        SwerveChassis.getInstance().resetAllEncoders();
+        //initializeSubsystems();
+        //SwerveChassis.getInstance().resetAllEncoders();
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/Robot.java:47]: remove //

File: src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/RunFunnelByJoystick.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+//    public void execute() {
+//        funnel.setPower(joystick.getAxisValue(SmartJoystick.Axis.RIGHT_X) * RobotConstants.General.SAFETY_POWER_CONVERSION_FACTOR);
+//    }
+//
+//    @Override
+//    public void end(boolean interrupted) {
+//        funnel.setPower(0);
+//    }
+//
+//}
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/RunFunnelByJoystick.java:25]: remove //

File: src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/RunFunnelByJoystick.java
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/RunFunnelByJoystick.java:1]: or remove file

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/neoIntake/NeoIntakeConstants.java [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 public class NeoIntakeConstants {
 
-    public static final int INTAKE_ID = 0;
+    public static final int INTAKE_ID = 12;
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/subsystems/Intake/neoIntake/NeoIntakeConstants.java:10]: `MOTOR_ID`

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/NeoRoller/NeoRoller.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.greenblitz.robotName.subsystems.arm.roller.IRoller;
+import edu.greenblitz.robotName.subsystems.arm.roller.RollerInputsAutoLogged;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+import static edu.greenblitz.robotName.subsystems.arm.roller.NeoRoller.NeoRollerConstants.*;
+import static edu.greenblitz.robotName.subsystems.shooter.Funnel.NeoFunnel.NeoFunnelConstants.BEAM_BREAKER_CHANNEL;
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/NeoRoller/NeoRoller.java:12]: dont import static constants its not cool

**Review by greenblitz4590 — COMMENTED** [review]
Yoni - 🤠👻

File: src/main/java/edu/greenblitz/robotName/OI.java
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/OI.java:1]: why in cr?

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollClockWise.java [hunk 6]
```diff
@@ -1,8 +1,9 @@
 package edu.greenblitz.robotName.commands.arm.roller;
 
 public class RollClockWise extends RollerCommand{
+
     @Override
-    public void initialize() {
+    public void execute() {
         roller.rollClockWise();
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollClockWise.java:7]:
```suggestion
        roller.rollClockwise();
```

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollCounterClockWise.java [hunk 7]
```diff
@@ -1,8 +1,9 @@
 package edu.greenblitz.robotName.commands.arm.roller;
 
 public class RollCounterClockWise extends RollerCommand{
+
     @Override
-    public void initialize() {
+    public void execute() {
         roller.rollCounterClockWise();
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollCounterClockWise.java:7]: same here W in lower case

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRollerCounterClockWiseUntilNoteIsInside.java [hunk 8]
```diff
@@ -1,8 +1,9 @@
 package edu.greenblitz.robotName.commands.arm.roller;
 
 public class RunRollerCounterClockWiseUntilNoteIsInside extends RollerCommand{
+
     @Override
-    public void initialize() {
+    public void execute() {
         roller.rollCounterClockWise();
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRollerCounterClockWiseUntilNoteIsInside.java:7]: same here

**Review by NoamRosenberg08 — CHANGES_REQUESTED** [review]

**Review by tnrz135 — APPROVED** [review]

**Review by NoamRosenberg08 — APPROVED** [review]

**Pull request merged**